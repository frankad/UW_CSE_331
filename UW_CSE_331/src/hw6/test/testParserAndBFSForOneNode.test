# test parser and BFS, a case for one node graph using
# graphWithOneNode dataset.

# build graph
LoadGraph graphWithOneNode graphWithOneNode.tsv

# test the graph is building process
ListNodes graphWithOneNode
ListChildren graphWithOneNode Eva
ListChildren graphWithOneNode David

# test BFS/findPathBFS
FindPath graphWithOneNode Eva Eva
FindPath graphWithOneNode Eva David
