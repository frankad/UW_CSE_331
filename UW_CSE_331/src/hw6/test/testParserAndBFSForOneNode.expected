# test parser and BFS, a case for one node graph using
# graphWithOneNode dataset.

# build graph
loaded graph graphWithOneNode

# test the graph is building process
graphWithOneNode contains: Eva
the children of Eva in graphWithOneNode are:
the children of David in graphWithOneNode are: node David is not found in the graph

# test BFS/findPathBFS
path from Eva to Eva:
unknown character David
